SU-HSDO/suhumsci
# Chore: Sparkbox Cleanup

# READY FOR REVIEW

## Summary
* update the pull request checklist and fix link
* remove old behat testing documentation
* update backstop instructions
* update backstop dependency
* update browserlist data and document

Related to [STN-753](https://sparkbox.atlassian.net/browse/STN-753)

## Need Review By (Date)
asap

## Urgency
medium

## Steps to Test
1. Delete your `node_modules` directory
2. Run `npm install`
3. Confirm the following:
    - [ ] `npm start` still runs
    - [ ] `npm run backstop:init`,  `npm run backstop:reference`,  and `npm run backstop:test` all run. You don't have to wait for them to complete, just confirm that they run
    - [ ] Updated documentation makes sense

## PR Checklist
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
- [Sparkbox PR Checklist](../docs/SparkboxPRChecklist.md)
